### ISSUE TITLE
Remote homology?

### ISSUE THREAD

**Issue by jasperhyp** [elttaes/Revisiting-PLMs#4]
Hi,  thanks for sharing the code!  I saw a remote homology folder in your repo.  Since I didn't notice you reported, what is that for?

**Comment by jasperhyp** [comment]
Also, is it possible to clean up the organization of directory, which seems not aligning with the content of the folders? (e.g. stability and contact are in .Function.antibiotic.esm.downstream_tape)

**Comment by jasperhyp** [comment]
I felt a bit lost...  Could you kindly provide some instructions on how I should run a benchmark test (whichever it is, say, any of `ABR`/`MIB`/`Flu`/`Sta`) with ESM-1b (pretrained), for example?

**Comment by elttaes** [comment]
Hi,
1. The result of the Remote Homology is on the last page of the Appendix.
2. The test code is already included in the training code, such as MIB(https://github.com/elttaes/Revisiting-PLMs/blob/main/Function/metal/esm/1b/train.py), ABR(https://github.com/elttaes/Revisiting-PLMs/blob/main/Function/antibiotic/esm/drug/drug_pretrain.py), Flu(https://github.com/elttaes/Revisiting-PLMs/blob/main/Fitness/flu/esm/flu/finetune_task_flu_pretrain.py) and Sta(https://github.com/elttaes/Revisiting-PLMs/blob/main/Fitness/sta/esm/sta/finetune_task_sta_ori.py).
I am so sorry for this bad organization, I will unify it as soon as possible.

**Comment by jasperhyp** [comment]
Thank you so much for your prompt and detailed response!  No worries at all, I read your paper with great pleasure.  Will check out the links.

**Comment by jasperhyp** [comment]
Hi!  I checked the training files and it looks like they use `.a3m` as data.  I think I can change them to the actual data files but just wanted to let you know.  Thanks!

**Comment by jasperhyp** [comment]
Also, the `test.txt` in `data/Antibiotic Resistance` contains 1344 sequences, which seem to differ from 530 as reported in the paper.  Can you clarify a bit?  Thanks!

**Comment by elttaes** [comment]
Thank you very much for reminding me that this part of the paper needs to be corrected.

**Comment by jasperhyp** [comment]
> Thank you very much for reminding me that this part of the paper needs to be corrected.

So the data have the correct numbers right?  Also, the loss in metal seems to be incorrect.  I think BCE is the one to be used compared to CE.  Thank you!
